### PR TITLE
Add frontend connection for timeSlot update endpoint

### PR DESCRIPTION
**Pull request by Jokauppi**
Related to Issue #

### Backend

- None

### Frontend

- Frontend query and calendar handler for updating a timeSlot was modified to use the API

## Expected Behaviour

- Changes made to timeslots in the frontend should be reflected in the database.
- Changes made to timeslots should retained in the timeslot calendar after making changes and e.g. changing the tab.

### CODE REVIEW

File: frontend/src/queries/timeSlots.ts [hunk 1]
```diff
@@ -1,24 +1,52 @@
...
-      (element) => parseInt(element.id, 10) === parseInt(timeSlot.id, 10),
-    )
-  ] = {
-    id: timeSlot.id,
-    start: timeSlot.start,
-    end: timeSlot.end,
-    editable: true,
+
+const addTimeSlot = async (newTimeSlot: any): Promise<void> => {
+  await fetch('api/timeslots/', {
```
Comment (vaisest) [inline frontend/src/queries/timeSlots.ts:9]: base path puuttuu?

File: frontend/src/queries/timeSlots.ts [hunk 2]
```diff
@@ -1,24 +1,52 @@
...
+  const response = await fetch(`${process.env.BASE_PATH}/api/timeslots/${id}`, {
+    method: 'DELETE',
+    headers: {
+      'Content-Type': 'application/json',
+    },
+  });
+  return response.text();
 };
 
+export default deleteTimeslot;
```
Comment (vaisest) [inline frontend/src/queries/timeSlots.ts:48]: Ylimääräinen export?

File: frontend/src/queries/timeSlots.ts (hunk 1 above)
Comment (Jokauppi) [reply frontend/src/queries/timeSlots.ts]: fixed

File: frontend/src/queries/timeSlots.ts (hunk 2 above)
Comment (Jokauppi) [reply frontend/src/queries/timeSlots.ts]: removed

**Review by vaisest — APPROVED** [review]

**Pull request merged**